include/simdjson/document.h

```diff
@@ -70,6 +70,8 @@ class tape_ref {
   inline size_t after_element() const noexcept;
   really_inline tape_type tape_ref_type() const noexcept;
   really_inline uint64_t tape_value() const noexcept;
+  really_inline uint32_t first_tape_value() const noexcept;
```

Comment:
When we do get to the end, these could use a comment as to their relationship to `tape_value()`.

Reply:
Yes. This is terrible naming and would be hard to debug if you don't understand the code base. I think first_tape_value should be "matching brace index" and second value should "size value", or something.

Reply:
Fixed in subsequent commit.